tracer/src/Datadog.Trace/IAST/Source.cs

```diff
@@ -9,7 +9,7 @@ namespace Datadog.Trace.Iast;
 
 internal readonly struct Source
 {
-    public Source(byte origin, string name, string value)
+    public Source(byte origin, string? name, string? value)
```

Comment:
Not sure name being nullable is a good thing? Is it needed because some parameters don't have names?

Reply:
Yes, the query string and the path of the query have no name. They have origin type which indicates the origin, but no name. I talked to the IAST team regarding this and it is not documented yet, but I was advised to follow this approach.